docs/js-core-1/week-2/lesson.md

```diff
@@ -25,6 +26,10 @@ By the end of this class, you should be able to:
 - Define comparator operators and list the different types
 - Use comparator operators in an `if` statement
 - Define a logical operator and give examples
+- Recognise and relate to different perspectives 
+- Recognise when conflict may arise 
+- Mitigate conflict in a productive manner
+- Resolve conflicts/disagreements 
```

Comment:
So I think that all these learning objectives are pretty hard to measure and some may be an overreach. _How_ can trainees show they can resolve conflicts and disagreements? Will they be brokering a new Good Friday Agreement? 😉 

If you can explain how they can do this, make that _how_ the LO. If you can't explain how they can do or show this, it's probably not an objective. It might be a course goal!

Reply:
Hi Sally, following with the comment below, we will take this offline and look it over. Right now, Esma is transferring the content. Well review all of these in the coming weeks, but let's make sure we have all content at least in one location.